modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/segregated/2d/2d-segregated-energy.i

```diff
@@ -0,0 +1,257 @@
+mu = 2.6
+rho = 1.0
+k = 5.0
+cp = 700
+alpha = 150
+advected_interp_method = 'average'
+velocity_interp_method = 'rc'
+
+pressure_tag = "pressure_grad"
+
+[Mesh]
+  [mesh]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = '0.3'
+    dy = '0.3'
+    ix = '3'
+    iy = '3'
+  []
+[]
+
+[GlobalParams]
+  rhie_chow_user_object = 'rc'
+[]
+
+[Problem]
+  nl_sys_names = 'u_system v_system pressure_system energy_system'
+  previous_nl_solution_required = true
+  error_on_jacobian_nonzero_reallocation = true
```

Comment:
Mostly commenting on this because I saw it actively excluded in your documentation. As this is the default for NS, you could remove it for your inputs